src/utils/mergeOptions.ts

```diff
@@ -205,6 +205,8 @@ function getInputOptions(
 		external: getExternal(config, command),
 		input: getOption('input'),
 		manualChunks: getOption('manualChunks'),
+		chunkGroupingSize: getOption('chunkGroupingSize'),
+		optimizeChunks: getOption('optimizeChunks'),
```

Comment:
I guess it would be more transparent to add the default value of 5000 here (as a second parameter to `getOption`) than to add it as a default parameter somewhere hidden in the optimization logic.